pkg/tkn/tkn.go

```diff
@@ -41,6 +41,8 @@ func AssertComponentVersion(version string, component string) {
 		actualVersion = cmd.MustSucceed("oc", "get", "tektonhub", "hub", "-o", "jsonpath={.status.version}").Stdout()
 	case "results":
 		actualVersion = cmd.MustSucceed("oc", "get", "tektonresult", "result", "-o", "jsonpath={.status.version}").Stdout()
+	case "manual-approval-gate":
+		actualVersion = cmd.MustSucceed("oc", "get", "manualapprovalgate", "manualapprovalgate", "-o", "jsonpath={.status.version}").Stdout()
```

Comment:
BTW, the name of resource must be `manual-approval-gate`, when you create a CR with different name you get a message like this
```
Ready: Resource ignored, Expected Name: manual-approval-gate, Got Name: manual-approval-gate-foo
```

Reply:
where should I place this CRD?